README.md

```diff
@@ -1,70 +1,220 @@
-# Getting Started with Create React App
+<a name="readme-top"></a>
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+<div align="center">
 
-## Available Scripts
+  <h1>Gladwin Tshepo Ramantso<h1>
```

Comment:
This seems to be someone else's name. I will advise you to either remove it entirely or change it to your name. I think removing it will be better though as it is confusing to see it there in addition to the project title.